VERSION

```diff
@@ -84,15 +84,15 @@ date="Unreleased developer copy"
 # Version numbers are described in the Libtool current:revision:age
 # format.
 
-libmpi_so_version=30:2:10
+libmpi_so_version=30:3:10
 libmpi_cxx_so_version=30:0:10
 libmpi_mpifh_so_version=31:1:11
 libmpi_usempi_tkr_so_version=30:1:10
 libmpi_usempi_ignore_tkr_so_version=30:1:10
-libmpi_usempif08_so_version=30:1:10
+libmpi_usempif08_so_version=30:2:10
```

Comment:
In addition to some other changes that are clearly in the F08 library, I notice that we changed the Fortran type of `MPI_ERRCODES_IGNORE` (which ends up in a header file that all the Fortran libraries include).  Does this mean we need to bump the R for all the Fortran libraries?

Reply:
I think so looking at the change to gni-mpi-mangling.pl